lib/services/role.go

```diff
@@ -493,6 +489,25 @@ func (r *RoleV3) CheckAndSetDefaults() error {
 	if r.Spec.Allow.Rules == nil {
 		r.Spec.Allow.Rules = CopyRulesSlice(AdminUserRules)
 	}
+	if r.Spec.Deny.Namespaces == nil {
+		r.Spec.Deny.Namespaces = []string{defaults.Namespace}
+	}
+
+	// check and correct the session ttl
+	maxSessionTTL, ok := r.Spec.Options.GetDuration(MaxSessionTTL)
```

Comment:
`ok` needs to be renamed to `err`. Golang has an idiom (getting value from a map) where `ok` as a 2nd parameter is a boolean, indicating that a value is found or not.